pkgs/applications/misc/athens/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "athens";
+  version = "2.0.0-beta.3";
+
+  src = fetchurl {
+    url = "https://github.com/athensresearch/athens/releases/download/v${version}/Athens-${version}.AppImage";
+    name = "Athens-${version}.AppImage";
+    sha256 = "sha256-2uw2ZGdUQLST+4fqDv/mrDKFuUhv543oZsDKkYFefrA=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
```

```suggestion
  appimageContents = appimageTools.extract { inherit pname version src; };
```
See https://github.com/NixOS/nixpkgs/blob/fd58133e8e526cf12cc2a8e23d22a1c81ce3f21b/pkgs/build-support/appimage/default.nix#L35-L38